### ISSUE TITLE
Crafting Table Crafting Broken for Server

### ISSUE THREAD

**Issue by Zunth5** [NillerMedDild/Enigmatica2Expert#454]
## Thank you for helping us improve. #####

_Please follow the issue template, and do not delete any part of it._
_Failure to comply will result in the issue being closed._

**Question or Issue**
Crafting Table Recipe does not resolve with any type of wood on Server. This works in single player

**Modpack Version**
1.46

**Additional Context**

**Comment by Zunth5** [comment]
Huh, this seems to effect a few items including vanilla doors and fences

**Comment by NillerMedDild** [comment]
So this error only occurs on a server? That sounds like the server isn't setup correctly.

**Comment by NillerMedDild** [comment]
That's just weird, but I'm fairly certain the issue only on your server.

I'm still not sure what your issue is. Are you unable to craft everything?

**Comment by Zunth5** [comment]
Yes, Unable to craft most things. It seems that when I run the server from the ServerStart.bat, it runs just fine. Running the Server out of AMP runs the server fine, aside from general crafting isn't working. This pack is the first I have seen with a crafting problem using AMP. Can you tell me what mod is being used to modify crafting recipes? 

Also, I cant seem to find the actual line in ServerStart.bat that actually initializes the server. I was hoping this would provide some insight into possible custom arguments that could be different than what AMP uses.

**Comment by NillerMedDild** [comment]
CraftTweaker is used to modify recipes, but not the ones you're having trouble with. Please refer to the ServerStart scripts issue tracker for questions relating to it.

**Comment by Zunth5** [comment]
I wanted to make a final note thanking you for the help. I found my answer here; https://oldforums.cubecoders.com/chat/amp-incompatible-with-spigot-paperspigot-mc-1-12
Turned out to be a services permissions issue because I moved the AMP directory to the default appdata location. Had to run the AMP server service as the user account that owned the appdata store. Thanks for the help and for the awesome pack.
